src/d3a/d3a_core/util.py

```diff
@@ -488,3 +488,7 @@ def convert_kilo_to_mega(unit_k):
 
 def convert_percent_to_ratio(unit_percent):
     return unit_percent / 100
+
+
+def short_offer_log_str(offer):
```

Comment:
Nice, may be this approach could also be used for unified/improved **logs** for OFFER/BID its forwarding, TRADE and its chaining. 
BUT future enhancement!

Reply:
Yeah, great, I just added it in the frame of 1758.